adoc/chapters/programming_interface.adoc

```diff
@@ -6671,10 +6671,8 @@ accessor()
   * All size queries return [code]#0#.
   * The return values of [code]#get_pointer()# and [code]#get_multi_ptr()# are
     unspecified.
-  * Trying to access the underlying memory is undefined behavior.
-
-A default constructed accessor can be passed to a <<sycl-kernel-function>>
-but it is not valid to register it with the command group handler.
+  * A default constructed accessor can be passed to a <<sycl-kernel-function>>,
```

Comment:
If you feel strongly about this, I don't mind adding the code-font.  However, we do not seem to use code font for every occurrence of class names, especially the common ones like "queue", "buffer", and "accessor".  For example, see the first sentence of this function description:

> Constructs an empty accessor which fulfills the following post-conditions

(There is no code-font for "accessor" here.)

I've actually been trying to tone-down the amount of code font in cases like this because I think all the font changes are a little distracting.  I don't really have a clear rule for when to use code font, but using it for every occurrence of a class name seems a bit too much.

Reply:
I don't feel strongly about this, so I'm happy leaving this as-is, maybe it's worth having a separate discussion about general conventions for formatting?